WordPress/src/main/java/org/wordpress/android/models/Person.java

```diff
@@ -0,0 +1,126 @@
+package org.wordpress.android.models;
+
+import android.support.annotation.Nullable;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.wordpress.android.util.AppLog;
+
+public class Person {
+    private long personID;
+    private String blogId;
+    private int localTableBlogId;
+
+    private String username;
+    private String firstName;
+    private String lastName;
+    private String displayName;
+    private String avatarUrl;
+    private String role;
+
+    public Person(long personID,
```

Comment:
I know this follows the way we construct a few other models in the app, but I'm really not a fan of a long list of parameters in the constructor. It's too easy to mess up the order of things.